libs/portals/my-pages/health/src/screens/Referrals/Referrals.graphql

```diff
@@ -0,0 +1,34 @@
+fragment Referral on HealthDirectorateReferral {
+  id
+  serviceName
+  validUntilDate
+  stateDisplay
+}
+
+query GetReferrals($locale: String) {
+  healthDirectorateReferrals(locale: $locale) {
+    referrals {
+      ...Referral
+    }
+  }
+}
+
+query GetReferralsDetail($locale: String) {
+  healthDirectorateReferrals(locale: $locale) {
+    referrals {
+      ...Referral
+      createdDate
+      reason
+      fromContactInfo {
+        name
+        profession
+        department
+      }
+      toContactInfo {
+        name
+        profession
+        department
+      }
+    }
```

Comment:
Maybe add a new endpoint, `healthDirectorateReferral` which accepts an `id`. Then the query only returns 1 result instead of a list which needs to be filtered.